documentation/asciidoc/computers/raspberry-pi/pcie.adoc

```diff
@@ -5,45 +5,28 @@ image::images/pcie.jpg[alt="Raspberry Pi connector for PCIe",width="70%"]
 
 Raspberry Pi 5 has an FPC connector on the right-hand side of the board. This connector breaks out a PCIe Gen 2.0 ×1 interface for fast peripherals.
 
-For more information about the PCIe FPC connector pinout and other details needed to create third-party devices, accessories, and HATs, see the https://datasheets.raspberrypi.com/pcie/pcie-connector-standard.pdf[Raspberry Pi Connector for PCIe] standards document. It should be read alongside the https://datasheets.raspberrypi.com/hat/hat-plus-specification.pdf[Raspberry Pi HAT+ Specification].
-
-=== Enabling PCIe
-
-By default the PCIe connector is not enabled. To enable it, add the following line to `/boot/firmware/config.txt`:
-
-[source]
-----
-dtparam=pciex1
-----
+To connect a PCIe https://datasheets.raspberrypi.com/hat/hat-plus-specification.pdf[HAT+ device], connect it to your Raspberry Pi. Your Raspberry Pi should automatically detect the device. To connect a non-HAT+ device, connect it to your Raspberry Pi, then <<enable-pcie, manually enable PCIe>>.
 
-Reboot with `sudo reboot` for the configuration changes to take effect.
-
-NOTE: You can also use the alias `nvme`.
+For more information about the PCIe FPC connector pinout and other details needed to create third-party devices, accessories, and HATs, see the https://datasheets.raspberrypi.com/pcie/pcie-connector-standard.pdf[Raspberry Pi Connector for PCIe] standards document. It should be read alongside the https://datasheets.raspberrypi.com/hat/hat-plus-specification.pdf[Raspberry Pi HAT+ Specification].
 
 NOTE: Enumeration of PCIe devices behind a switch is https://github.com/raspberrypi/firmware/issues/1833[not currently supported].
 
-=== PCIe Gen 3.0
+=== Enable PCIe
 
-The connection is certified for Gen 2.0 speeds (5 GT/sec), but you can force Gen 3.0 (10 GT/sec) speeds if you add the following lines to your `/boot/firmware/config.txt`.
+By default, the PCIe connector is not enabled unless connected to a HAT+ device. To enable the connector, add the following line to `/boot/firmware/config.txt`:
 
-[source]
+[source,perl]
```

Comment:
Why is this tagged as perl?

Reply:
This syntax doesn't have a reasonable highlighting equivalent in our toolbox, so I just chose something that renders equal signs a different color so it's slightly easier to read large blocks of text with equal signs in between. Perl was a slightly arbitrary choice, but I tested bash and a couple of others and the square bracket `[]` syntax seems to mess up the highlighting. Perl, as we know, tolerates just about any string of characters without breaking, so I went with that 😄